assets/js/util/index.js

```diff
@@ -714,28 +714,49 @@ export const deleteCache = ( cacheType, cacheKey ) => {
 	return true;
 };
 
-export const findTagInIframeContent = ( iframe, module ) => {
+export const findTagInHtmlContent = ( html, module ) => {
 	let existingTag = false;
 
-	if ( ! iframe ) {
+	if ( ! html ) {
 		return false;
 	}
 
-	// Check if tag present in <head>.
-	const head = iframe.contentWindow.document.querySelector( 'head' );
-	if ( head ) {
-		existingTag = extractTag( head.innerHTML, module );
-	}
+	existingTag = extractTag( html, module );
+
+	return existingTag;
+};
+
+/**
+ * Looks for existing tag requesting front end html, if no existing tag was found on server side
+ * while requesting list of accounts.
+ *
+ * @param {string} module Module slug.
+ */
+export const getExistingTag = async( module ) => {
+
+	try {
+		let tagFound = data.getCache( module, 'existingTag', 300 );
+
+		if ( false === tagFound ) {
+			const html = await fetch( `${googlesitekit.admin.siteURL}?tagverify=1&timestamp=${Date.now()}` ).then( res => {
```

Comment:
This is using the Site Kit specific "site URL", which may be filtered in development setups. It should use the actual home URL, like it did in the iframe.
Using the URL of another site will possibly cause CORS issues.

Reply:
changed https://github.com/google/site-kit-wp/pull/82/commits/f3c0389e25dd95eb8dc667eb00a9db297bed639c